docs/imaging/sync.rst

```diff
@@ -4,31 +4,117 @@
+The *outcome* of that synchronisation stage should more properly be called time information, but throughout the Imaging pipeline is often *also* called synchronisation data. 
+
+Three methods are currently in use within the Moser group:
+
+* `Wavesurfer <http://wavesurfer.janelia.org/>`_ with a National Instruments GPIO card
+* Femtonics Mesc synchronisation
+* ScanImage Presynchronisation
 
-Ingest of sync data happens in the ``Sync`` table (see /dj_schemas/sync.py). For this, sync data is extracted according to dataset type (*Wavesurfer* vs. *MESC* vs. ...). Each experiment type (e.g. *femtonics*, *2Pminiscope_A*) has an entry under ``Setup``, which holds at least the *polarity* and for Wavesurfer files also info about which stream served as master and what tolerance to allow between the end of sync streams. The ``Naming`` part table allows downstream processes to infer what specific sync streams were called and to retrieve them accordingly. For example, if a table requires sync data for rotary (wheel) tracking data, it will query the ``Naming`` table to retrieve the name of the sync stream for that particular setup. That sync stream can then be retrieved from the main ``Sync`` table. 
+Each of these implementation is then inserted into a very generalised schema, which maps the time data from each time series into a single temporal frame of reference. This generalised approach has both benefits and drawbacks: the primary benefit is that it can be extended to new types of data - and new hardware - very easily. The primary drawback is that it adds complexity and overhead to common operations, and that the users need to be aware of that complexity. 
 
+Synchronisation data is stored in the ``Sync`` table and its sub-tables. 
 
-.. figure:: /_static/imaging/sync_tables.PNG
+.. figure:: /_static/imaging/sync/sync_tables.PNG
    :alt: Synchronisation section of the imaging schema
 
+Each Session is generated by a **Setup** (:ref:`Imaging Terminology Setup`), which is tracked in the ``Setup`` table. Each entry in the ``Setup`` table has a list of the types of synchronisation data (and how to interpret it) that might be generated in the table ``Setup.Sync``.::
+
+  Setup.Sync & {"setup_name":"Mini Blue", "setup_timestamp":"2020-08-20 00:00:00"}
+
+.. figure:: /_static/imaging/sync/setup-sync-table.png
+   :alt: Contents of the ``Setup.Sync`` table
+
+
+When a new Session is ingested, the synchronisation data is inserted into the ``Sync`` table and its part tables. Each Session has one (or more) entries in the ``Sync`` table, corresponding to a single time series, from a single instrument, with each time series being identified by a ``sync_name``.
+
+The actual timestamps themselves are stored in the column ``sync_data`` of the ``Sync`` table. The values are stored in *samples*. The sample rate (samples per second) is also stored, in column ``sample_rate``, if you wish to convert into time periods in seconds. 
+
```

Comment:
Add: Note that all sync data that is recorded by scanimage itself is in seconds already (`sample_rate = 1`).